vendor/github.com/ActiveState/gozip/LICENSE.txt

```diff
@@ -0,0 +1,5 @@
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
I thought we would fork the repository into its own space like we did for Pillow.  I don't think we should be forking this into TheHomeRepot.

Reply:
I'm wondering if this is just the way DX is doing things, but I will happily check, having two copies isn't necessarily a great idea.

Reply:
Were you unable to use https://github.com/ActiveState/cli/blob/master/internal/unarchiver/zip.go ?

I'd rather we update that to add the functionality you need, rather than add a new dependency.

Reply:
I can look at the code to use the internal unzip code, and will copy across what is needed from Gozip, as needed. I expect that things should just work for the most part without needing to copy anything. Gozip is just a thin eggshell over the internal Go zip library anyway.

Reply:
I changed the custom-installer code to use the basic Golang primitives instead of using the Gozip library.